packages/dcos-integration-test/extra/test_networking.py

```diff
@@ -14,8 +14,9 @@
 from dcos_test_utils import marathon
```

Comment:
Could you please move all non-calico changes to a separate PR? it's impossible to review. I spent the last two years to stabilize it, you can find all the default in git log. We hit lots of extremely tricky issues, few kernel bugs, etc. I don't feel safe to review or merge a 811-line patch. It's really hard to understand why certain changes have been made in this PR, it's just one huge commit for `test_networking.py`. BTW I'm not quite understand why the python style has been changed, it does not match with the style in all other modules from dc/os test suites.